apps/authentication-service/cypress/e2e/express.cy.ts

```diff
@@ -0,0 +1,21 @@
+describe("Express", () => {
+  const PORT: number = 8089;
+
+  it("returns pong", () => {
+    cy.request(`http://localhost:${PORT}/ping`).then(
+      (response: Cypress.Response<{ message: string }>) => {
+        expect(response.status).to.eq(200);
+        expect(response.body.message).to.eq("pong");
+      },
+    );
+  });
+
+  it("handles not found", () => {
+    cy.request({
+      url: `http://localhost:${PORT}/pig`,
+      failOnStatusCode: false,
+    })
+      .its("status")
+      .should("eq", 404);
+  });
+});
```

Comment:
New file that includes the basic expressjs tests. These were pulled out of the rateLimit e2e test file.